aerialist/px4/obstacle.py

```diff
@@ -124,15 +146,29 @@ def to_dict(self):
                 "x": self.position.x,
                 "y": self.position.y,
                 "z": self.position.z,
-                "r": self.position.r,
+                "angle": self.position.r,
             },
+            "shape": self.shape,
         }
 
     @classmethod
     def distance_to_many(cls, obstacles: List[Obstacle], line: LineString):
         boxes = [o.geometry for o in obstacles]
         dist = min([sum([b.distance(Point(*p)) for b in boxes]) for p in line.coords])
-        return dist
+        dist_list = [sum([b.distance(Point(*p)) for b in boxes]) for p in line.coords]
+        dist_avg = sum(dist_list) / len(dist_list)
+        return dist, dist_list
```

Comment:
adding new return values to this function could break current code, let's check together

Reply:
I will create a new method.